lib/dpul_collections/indexing_pipeline/figgy/hydration_consumer.ex

```diff
@@ -135,7 +135,7 @@ defmodule DpulCollections.IndexingPipeline.Figgy.HydrationConsumer do
         %{cache_version: cache_version}
       )
       when internal_resource in ["DeletionMarker"] and
-             resource_type in ["EphemeraFolder", "EphemeraTerm"] do
+             resource_type in ["EphemeraFolder", "EphemeraTerm", "EphemeraProject", "EphemeraBox"] do
```

Comment:
Why are we hydrating boxes and folders?

Reply:
It was an early iteration of the PR that I decided to leave in place. I'm thinking that it might be necessary for https://github.com/pulibrary/dpul-collections/issues/75. EphemeraTerms are also hydrated. We're leaving them in for the same reason, I suppose.

Reply:
@eliotjordan I don't think we're going to need it for #75. I don't think we need Terms either. Should we talk about it synchronously so I can understand better where you're comin' from?

Reply:
It's OK. I'll remove it and make a ticket for removing hydration of Terms.

Reply:
https://github.com/pulibrary/dpul-collections/issues/481